circle.yml

```diff
@@ -196,16 +207,49 @@ jobs:
       - *build
       - *save-deps-cache
       - *store-package
+      - *add-package-to-workspace
       - *test
       - *store-randomcode-seed
 
+  # The job collects built packages (available in the workspace) and
+  # uploads them to the GitHub release matching the git tag.
+  deploy:
+    docker:
+      - image: circleci/golang
+    steps:
+      - run:
+          name: "Install ghr"
+          command: go get -u github.com/tcnksm/ghr
+      - attach_workspace:
+          at: ~/package
+      - run:
+          name: "Create GitHub release"
+          command: |
+            prerelease_flag=$([[ $CIRCLE_TAG =~ ^v[0-9\.]+$ ]] || echo '-prerelease')
+            ghr -u ethereum -r cpp-ethereum $prerelease_flag $CIRCLE_TAG ~/package
```

Comment:
Just checked github.com/tcnksm/ghr, nice.

I wonder what the "prerelease" looks like.

Reply:
Oh its that red icon. In that case I think having commit hash is very useful, but could be done in a separate PR.